mappings/net/minecraft/container/SmithingTableContainer.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_4862 net/minecraft/container/SmithingTableContainer
+	FIELD field_22485 smithingRecipes Ljava/util/Map;
```

Comment:
static final field

Reply:
Fixed in 090296774ccd61114f11805cad4e925942a74744.